src/dotnet-interactive-vscode/src/tests/unit/testClientAdapter.ts

```diff
@@ -0,0 +1,25 @@
+import { Observable } from "rxjs";
```

Comment:
@brettfo is this right? I had this after updating master with your latest merge.

Reply:
After my last PR `testClientAdapter.ts` looks like this:

``` typescript
import { Observable } from "rxjs";
import { EventEnvelope } from "../../events";
import { Command } from "../../commands";

// Replays all events given to it
export class TestClientTransport {
    constructor(readonly fakedEvents: { [key: string]: {eventType: string, event: any}[] }) {
    }

    submitCommand(commandType: string, command: Command): Observable<EventEnvelope> {
        let eventsToReturn = this.fakedEvents[commandType];
        return new Observable<EventEnvelope>(subscriber => {
            for (let event of eventsToReturn) {
                subscriber.next({
                    eventType: event.eventType,
                    event: event.event,
                    cause: {
                        token: ''
                    }
                });
            }

            subscriber.complete();
        });
    }
}
```

Reply:
And the location looks correct, too.

Reply:
@brettfo  so the file is consistent with master.